File: app/src/main/java/com/cyl/musiclake/musicApi/PlaylistInfo.java
package com.cyl.musiclake.musicApi;

import com.google.gson.annotations.SerializedName;

/**
 * Created by master on 2018/4/5.
 */

public class PlaylistInfo {

    @SerializedName("name")
    String name;

    public PlaylistInfo(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: musicapi/src/main/java/com/cyl/musicapi/playlist/PlaylistInfo.kt
package com.cyl.musicapi.playlist


import com.google.gson.annotations.SerializedName

data class PlaylistInfo(
        @SerializedName("name")
        val name: String = "",
        @SerializedName("description")
        val description: String? = null,
        @SerializedName("cover")
        val cover: String? = null,
        @SerializedName("playCount")
        val playCount: Long = 0,
        @SerializedName("id")
        val id: String = "",
        @SerializedName("total")
        val total: Int = 0,
        @SerializedName("list")
        val list: MutableList<MusicInfo>? = null)


